pages/blog.js

```diff
@@ -28,7 +43,33 @@ class Blog extends React.Component {
 
     return { posts: json };
   }
+
+  getGategoriesList(url) {
+    const { posts } = this.props;
+    const chosenCategory = url.query.chosen;
+    const categories = posts
+      .map(post => post.tags)
+      .reduce((a, b) => a.concat(Array.isArray(b) ? flatten(b) : b), []); // flatten
+    const uniqCategories = [...new Set(categories)];
+
+    const selectedPosts = chosenCategory && chosenCategory !== 'All'
```

Comment:
Query parans never equals `all`. Yes, it should. But won't.

Reply:
left from the old version I removed `&& chosenCategory !== 'All'`